### PR TITLE
增加category-analytics

### PR DESCRIPTION
**Pull request by degfw**

### CODE REVIEW

**Comment by IceCodeNew** [comment]
If these domains are sorted out by others or referring to internet sources, please label them out.

**Comment by degfw** [comment]
@IceCodeNew I'm sorry for so long to reply to you.

For these domains, `in.appcenter.ms` and `app-measurement.com` are found from my device, and others were collecting from GitHub. 

However, I couldn't re-find them all at once, so I can't referring their sources all here.

**Comment by IceCodeNew** [comment]
> @IceCodeNew I'm sorry for so long to reply to you.
> 
> For these domains, `in.appcenter.ms` and `app-measurement.com` are found from my device, and others were collecting from GitHub.
> 
> However, I couldn't re-find them all at once, so I can't referring their sources all here.

Then you should only submit domains that you are sure they are utilizing ONLY for tracking.

File: data/category-analytics [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# List of all domains being used in log collection or monitor.
+
+include:category-ads
```
Comment (IceCodeNew) [inline data/category-analytics:3]: should not include `ads` as they are completely different use cases.

File: data/category-analytics [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+aax-us-east.amazon-adsystem.com
+alb.reddit.com
+analytics.brave.com
+analytics.map.qq.com
+analytics.plex.tv
+app-measurement.com
+be-nl.norton.com
+c.s-microsoft.com
```
Comment (IceCodeNew) [inline data/category-analytics]: If providing the referring sources turns to be difficult and painful, you could just concatenate all the domain lists that you have accessed during doing this PR, then give me the all-in-one list for me to do the review.

**Review by IceCodeNew — CHANGES_REQUESTED** [review]
If providing the referring sources turns to be difficult and painful, you could just concatenate all the domain lists that you have accessed during doing this PR, then give me the all-in-one list for me to do the review.

**Comment by degfw** [comment]
The only domains I can find from my device are the three left, and they are used within Wechat, Clash, and Google's app.

**Comment by IceCodeNew** [comment]
> The only domains I can find from my device are the three left, and they are used within Wechat, Clash, and Google's app.

I see. For approving these changes, may I ask how do you determine that these domains are for analytics purposes?

File: data/category-analytics [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# List of all domains being used in log collection or monitor.
+
+analytics.map.qq.com
+app-measurement.com
+in.appcenter.ms
```
Comment (KevinZonda) [inline data/category-analytics:5]: Would you mind providing this domain's purpose? I found it should be Microsoft App Center SDK. So it should not be kindly analysis tracker?

**Review by KevinZonda — CHANGES_REQUESTED** [review]

**Comment by KevinZonda** [comment]
app-measurement.com is for firebase. [refer](app-measurement.com)  
analytics.map.qq.com is for Tencent Map /cc @IceCodeNew

**Comment by Loyalsoldier** [comment]
Due to the lack of manpower, I personally recommend not collecting or adding "advertising" or "analytics" domains to the project. Instead we'd better use mature third-party projects that are compatible with MIT license.

Or let users generate their own geosite files after we refactor the project. I'd like to do it when I'm free lately.

**Comment by IceCodeNew** [comment]
> Due to the lack of manpower, I personally recommend not collecting or adding "advertising" or "analytics" domains to the project. Instead we'd better use mature third-party projects that are compatible with MIT license.
> 
> Or let users generate their own geosite files after we refactor the project. I'd like to do it when I'm free lately.

I totally agree with that.

**Comment by KevinZonda** [comment]
> Due to the lack of manpower, I personally recommend not collecting or adding "advertising" or "analytics" domains to the project. Instead we'd better use mature third-party projects that are compatible with MIT license.
> 
> Or let users generate their own geosite files after we refactor the project. I'd like to do it when I'm free lately.

It could be a better choice, totally agree with this.

So, should we close this issue? /cc @IceCodeNew @Loyalsoldier

**Pull request closed without merging**